src/QSymbolicsBase/basic_ops_homogeneous.jl

```diff
@@ -45,7 +45,7 @@ function Base.show(io::IO, x::SScaledKet)
 end
 const SScaledOperator = SScaled{AbstractOperator}
 function Base.show(io::IO, x::SScaledOperator)
-    if x.coeff isa Number
+    if x.coeff isa Real
```

Comment:
Fix for #54.

Reply:
https://github.com/QuantumSavory/QuantumSymbolics.jl/issues/54#issuecomment-2187330143